app/models/user.rb

```diff
@@ -487,6 +487,7 @@ def email_hash
 
   def reload
     @unread_notifications = nil
+    @all_unread_notifications = nil
```

Comment:
Sorry can we rename the `unread_notifications` and `unread_total_notifications` variables too? I looked into each method and the variable names right now do not reflect the nuances of each count. 

Also I wonder if the variable name should be `all_unread_notifications_count` instead to better reflect that it is actually a count and not an array of notifications.

Reply:
Renaming the existing variables is probably going to break some themes and plugins 😢 Maybe we should deprecated them for now?

But will rename the new variable to `all_unread_notifications_count`.

Reply:
> But will rename the new variable to all_unread_notifications_count. 

Sounds good to me :+1: